draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -321,15 +321,15 @@
 	  using COSE_Encrypt or COSE_Encrypt0.
 	</t>
 	<t>
-	  The following example (using JSON notation) illustrates a symmetric key that could subsequently be
+	  The following example (using CBOR diagnostic notation) illustrates a symmetric key that could subsequently be
 	  encrypted for use in the <spanx style="verb">Encrypted_COSE_Key</spanx> member:
 	</t>
 	<figure>
 	  <artwork><![CDATA[
-  {
-   "kty": "oct",
-   "alg": "HS256",
-   "k": "ZoRSOrFzN_FzUA5XKMYoVHyzff5oRJxl-IXRtztJ6uE"
+  {  
+   /kty/ 1 : /Symmetric/ 4,
+   /alg/ 3 : /HMAC256/ 5,
```

Comment:
I'd change "HMAC256" to "HMAC w/ SHA-256" - matching the description in the registry at https://www.iana.org/assignments/cose/cose.xhtml.

Reply:
Ok